NU3005 warning on AppVeyor

See log: https://ci.appveyor.com/project/nlog/nlog-framework-logging/builds/38390011

>  warning NU3005: Package '...' from source 'C:\Program Files\dotnet\sdk\NuGetFallbackFolder': The package signature file entry i

related: https://github.com/NLog/NLog.Web/pull/430

--- a/test/NLog.Extensions.Hosting.Tests/NLog.Extensions.Hosting.Tests.csproj
+++ b/test/NLog.Extensions.Hosting.Tests/NLog.Extensions.Hosting.Tests.csproj
@@ -13,6 +13,7 @@
     <AssemblyOriginatorKeyFile>..\..\src\NLog.snk</AssemblyOriginatorKeyFile>
     <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
     <LangVersion>latest</LangVersion>
+    <DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>
   </PropertyGroup>
   
   <ItemGroup  Condition=" '$(TargetFramework)' == 'netcoreapp2.1' ">

--- a/test/NLog.Extensions.Logging.Tests/NLog.Extensions.Logging.Tests.csproj
+++ b/test/NLog.Extensions.Logging.Tests/NLog.Extensions.Logging.Tests.csproj
@@ -13,6 +13,7 @@
     <AssemblyOriginatorKeyFile>..\..\src\NLog.snk</AssemblyOriginatorKeyFile>
     <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
     <LangVersion>latest</LangVersion>
+    <DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>
   </PropertyGroup>
 
   <PropertyGroup Condition=" '$(TargetFramework)' == 'netcoreapp1.1' or '$(TargetFramework)' == 'net452' ">

